src/HelloSubscriber/Program.cs

```diff
@@ -1,4 +1,20 @@
-var app = WebApplication.CreateBuilder(args).Build();
+using Serilog;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Configuration.AddJsonFile("appsettings.json", optional: true, reloadOnChange: true);
+builder.Configuration.AddEnvironmentVariables();
+builder.Configuration.AddCommandLine(args);
+
+// Serilog provides a more concise console logging experience with colored tokens
+builder.Host.UseSerilog((context, configuration) =>
+{
+    configuration.ReadFrom.Configuration(context.Configuration);
```

Comment:
This line doesn't work as expected with the current appsettings.json, as the Logging configuration section is not used by Serilog